js/google_ima.js

```diff
@@ -425,7 +425,8 @@ require("../html5-common/js/utils/utils.js");
           //displaying it, so just notify AMC that we are showing an overlay.
           else
           {
-            _amc.notifyNonlinearAdStarted(this.currentAMCAdPod.id);
+            // raise WILL_PLAY_NONLINEAR_AD event and notify nonlinear ad started
+            _amc.sendURLToLoadAndPlayNonLinearAd(this.currentAMCAdPod, this.currentAMCAdPod.id, this.currentAMCAdPod.ad.tag_url);
```

Comment:
I see, thanks for the explanation!

On Tue, Dec 1, 2015 at 3:50 PM, gfrankOoyala <notifications@github.com>
wrote:

> In js/google_ima.js
> <https://github.com/ooyala/html5-ad-plugins/pull/16#discussion_r46358278>:
>
> > @@ -425,7 +425,8 @@ require("../html5-common/js/utils/utils.js");
> >            //displaying it, so just notify AMC that we are showing an overlay.
> >            else
> >            {
> > -            _amc.notifyNonlinearAdStarted(this.currentAMCAdPod.id);
> > +            // raise WILL_PLAY_NONLINEAR_AD event and notify nonlinear ad started
> > +            _amc.sendURLToLoadAndPlayNonLinearAd(this.currentAMCAdPod, this.currentAMCAdPod.id, this.currentAMCAdPod.ad.tag_url);
>
> If the url is there, then it will send that to the player, which it will
> use to render the overlay. In VAST it will render the image that i sent via
> the streamurl. So it is best to send null if it isn't going to render it by
> the player but by the sdk to avoid any confusion or mistakes.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ooyala/html5-ad-plugins/pull/16/files#r46358278>.
>